test/functional/users_controller_test.rb

```diff
@@ -257,4 +257,57 @@ def setup
     post :test_digest_email
     assert_redirected_to '/'
   end
+
+  test 'Oauth tags are in form of oauth:provider for github' do
+    UserSession.create(users(:jeff))
+    user = users(:jeff)
+    auth = { "provider" => "github", "uid" => "123456789"}
+    uid = user.id
+    identity = UserTag.create_with_omniauth(auth, uid)
+    get :profile
+    assert_response :redirect
+    assert_redirected_to '/profile/jeff'
+    s = "span#tag_"+identity.id.to_s
+    assert_select s,'omniauth:github'
```

Comment:
@jywarren can you please tell me the what I am doing wrong here. I checked in the view it is https://github.com/publiclab/plots2/blob/de4d09faf10d16ab2f15291373ca954ebaba60ee/app/views/tag/_tags.html.erb#L20-L25
It says span with id as tag_+tag.id. Is tag.id not same as usertag.id?

Reply:
Hmm, can you check it locally? Maybe also look for the total count of items like `span.label` to be sure they're appearing at all?

Reply:
Yeah I am trying to debug this. I am trying my level best to write these test asap. It is giving errors.

On localhost everything is working fine. Also in production things are working fine. Just trying to debug the tests.